### PR TITLE
Tasks for downloading PG&E data

### PR DESCRIPTION
**Pull request by peuanthinsan**

### CODE REVIEW

File: src/containers/dashboards/index.js [hunk 1]
```diff
@@ -54,9 +54,12 @@ const Dashboards = ({
 }) => {
   const currentDashboardName = currentDashboard in dashboards ? dashboards[currentDashboard].name : '';
 
+  const PGEStartDate = new Date('6/05/20');
+  const PGEEndDate = new Date('7/05/20');
```
Comment (peuanthinsan) [inline src/containers/dashboards/index.js:58]: This part needs to be tied to the script stop date

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,7 +1,22 @@
 version: 2.1
 orbs:
   node: circleci/node@2.1.1
+  python: circleci/python@0.3.2
 jobs:
+  downloadPGEData:
+    docker:
+      - image: circleci/python:latest
+    environment:
+      - PGE_STOP_DATE=7/5/20
```
Comment (peuanthinsan) [inline .circleci/config.yml]: This part must be manually edited... unless perhaps we allow for errors to occur and search until 12/31/20 but that's not ideal

File: scripts/pgande.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        stop = datetime.strptime(stop,date_format)
+    res1 = pd.DataFrame()
+    for date in daterange(start,stop):
+        datename = date.strftime('%Y%m%d')
+        if show_progress:
+            print(f"Processing {date}...",flush=True)
+        try:
+            df = None
+            df = get_load_profile(date)
+            df.to_csv(f'public/api/pge/{datename}.csv')
```
Comment (peuanthinsan) [inline scripts/pgande.py:59]: This was the modification made to original script

\+ line 73

File: .circleci/config.yml (hunk 2 above)
Comment (krohling) [reply .circleci/config.yml]: @peuanthinsan can you undo your changes to this file?

File: scripts/pgande.py (hunk 3 above)
Comment (krohling) [reply scripts/pgande.py:59]: Is "df = None" necessary?

**Review by krohling — APPROVED** [review]

File: scripts/pgande.py (hunk 3 above)
Comment (peuanthinsan) [reply scripts/pgande.py:59]: original code. I can just fix that up np

**Pull request merged**